packages/image_picker/image_picker_ios/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 0.8.5+1
 
 * Removes unnecessary imports.
+* Fixes 'messages.g.h' file not found.
```

Comment:
You've merged incorrectly here; you're now adding to the change list for a version that has already been published. You need a new version bump.